.clang-tidy

```diff
@@ -25,25 +31,31 @@ Checks: >
   -hicpp-special-member-functions,
   -hicpp-use-equals-default,
   -hicpp-vararg,
-  -hicpp-vararg,
   -llvm-header-guard,
   -llvm-include-order,
+  -llvm-qualified-auto,
   -llvmlibc-*,
   -misc-no-recursion,
-  -misc-no-recursion,
```

Comment:
Adding this would give a lot of warnings since we use recursion in many places , no ?